django/test/testcases.py

```diff
@@ -1345,3 +1351,26 @@ def setUpClass(cls):
     def tearDownClass(cls):
         super().tearDownClass()
         cls._lockfile.close()
+
+
+class WidgetTestCase(SimpleTestCase):
+    """
+    For testing widget rendering in Django's test suite (not a public API).
+    """
```

Comment:
For now in https://github.com/django/django/pull/8401, I've done `from forms_tests.widget_tests.base import WidgetTest` in `postgres_tests`. I don't think it causes any test isolation issues -- can you think of any problems?